js/ui/file_manager/ui.file_manager.js

```diff
@@ -419,6 +419,24 @@ class FileManager extends Widget {
         });
     }
 
+    /**
+     * @name dxFileManagetMethods.setPath
+     * @publicName setPath(path)
+     * @return Promise<void>
+     */
+    setPath(path) {
+        return this._controller.setCurrentPath(path);
+    }
+
+    /**
+     * @name dxFileManagetMethods.getPath
+     * @publicName getPath()
+     * @return string
+     */
+    getPath() {
+        return this._controller.getCurrentPath();
+    }
+
```

Comment:
How about `"initialPath"` option? This option will open the specified folder immediately without loading root folder content into the item list view.

I think it's better to use `"path"` option instead of the `getPath()/setPath()` methods. So we will be able to replace `"initialPath"` with this `"path"` option.

Reply:
I think about it, but the methods approach coverages more scenarios. For instance,

- How to complete something, after change folder? Event?
- How to get current path? In your case, we'll update options object

Reply:
> How to complete something, after change folder? Event?

We need `currentFolderChanged` and `errorOccured` events regardless of the current PR. So, this events may be a solution for this case too.

> How to get current path? In your case, we'll update options object

`fileManager.option("path")`
AFAIK we shouldn't have problems with this aproach.

Reply:
So, I have implemented your suggestions. What do you think about this version?

Reply:
Not really. In my previous comment I mentioned the `currentFolderChanged` event, not the `pathChanged`, because:
- `currentFolderChanged` event gives full information about the current folder - it's id, path, lastWriteTime, in future - metadata and access rights
- removes the problem when the folder changed from one to another with the same name (relative path), so the path will not change, but the folder will change.

By the way, we can address `currentFolderChanged` event implementation in the next PR.

Reply:
onPathChanged -> onCurrentDirectoryChanged

Reply:
OK